ui/pages/error-page/index.scss

```diff
@@ -29,7 +29,7 @@
 
 // override styles for sentry feedback form
 #sentry-feedback {
-  --font-family: var(--font-family-sans);
+  --font-family: var(--font-family-default);
```

Comment:
Updating font family CSS variable based on [migration docs](https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/MIGRATION.md#from-version-600-to-700)